app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -34,6 +34,13 @@ def phase
     phase_from_keys 'claimPhaseDates', 'latestPhaseType'
   end
 
+  def files_needed
+    list_objects_with_key(
+      %w(claimTrackedItems stillNeedFromYouList),
+      %w(uploadsAllowed)
```

Comment:
`uploadsAllowed` looks to be a boolean, do you want to check its value rather than just its presence?